seda.muradyan/repo/exercise_03/exercise_03.py

```diff
@@ -0,0 +1,9 @@
+#a
+print("*\n**\n***\n****")
```

Comment:
LW: What if you need to print more than 4 lines of asterisks?

Hint: Use multiplication of character.